File: storage/data_transpiled/at_coder/arc019/A/4241718.java
public static int input ( ) {
  String A = input ( ) ;
  String str1 = A . replace ( 'O' , '0' ) ;
  String str2 = str1 . replace ( 'D' , '0' ) ;
  String str3 = str2 . replace ( 'I' , '1' ) ;
  String str4 = str3 . replace ( 'Z' , '2' ) ;
  String str5 = str4 . replace ( 'S' , '5' ) ;
  String str6 = str5 . replace ( 'B' , '8' ) ;
  return ( int ) ( str6 ) ;
}

File: storage/data_original/at_coder/arc019/A/4241718.py
A = input()
str1 = A.replace('O','0')
str2 = str1.replace('D','0')  
str3 = str2.replace('I','1')
str4 = str3.replace('Z','2')
str5 = str4.replace('S','5')
str6 = str5.replace('B','8')
print(int(str6))